documentation/docs/developers/deployments.md

```diff
@@ -142,9 +143,72 @@ If for any reason the local content repo gets into a bad/conflicted state, it ca
 The content repo can be opened via shortcut `yarn workflow repo open`
 
 
+## Android Management
+For deployments that are intended to be published to the Google Play Store, some additional configuration is required.
 
+### App ID and App Name
+The App ID uniquely identifies an Android app on devices and in the Google Play Store (see the official [Android developer docs](https://developer.android.com/build/configure-app-module#set-application-id) for more details). The App Name is the name that will be displayed to users on devices and in the Google Play Store. These values must be set in the deployment config before the app can be built as an Android bundle.
+
```

Comment:
nit(non-blocking) - it might be worth emphasisisng the section in the android docs that the app id must contain at least 2 segments and that the name cannot be changed after without creating a new app.

https://developer.android.com/build/configure-app-module

Reply:
Good suggestion, I've incorporated this and updated the PR description